commons/connector/upload/altoida/altoida_assessments_summary.avsc

```diff
@@ -0,0 +1,15 @@
+{
+    "namespace": "org.radarcns.connector.upload.altoida",
+    "name": "AltoidaAssessmentsSummary",
+    "type": "record",
+    "doc": "Summary timestamps of a completed Altoida assessment.",
+    "fields": [
+      { "name": "time", "type": "double", "doc": "Timestamp in UTC (s) when the assessment is started by the subject." },
+      { "name": "timeReceived", "type": "double", "doc": "Timestamp in unix time received from Altoida API."},
+      { "name": "assessmentName", "type": ["null", "string"], "doc": "Assessment name.", "default": null },
+      { "name": "startTime", "type": "double", "doc": "Timestamp in UTC (s) when the assessment is started." },
```

Comment:
The description is identical to “time”, is there a difference?

Reply:
Yeah they are identical. I can remove `time`.